pyant/plotting.py

```diff
@@ -233,18 +233,22 @@ def gain_heatmap(beam, polarization=None, resolution=201, min_elevation=0.0, lev
     SdB[np.isinf(SdB)] = 0
     SdB[np.isnan(SdB)] = 0
     SdB[SdB < 0] = 0
-    conf = ax.contourf(K[:,:,0], K[:,:,1], SdB, cmap=cm.plasma, vmin=0, vmax=np.max(SdB), levels=levels)
+    # conf = ax.contourf(K[:,:,0], K[:,:,1], SdB, cmap=cm.plasma, vmin=0, vmax=np.max(SdB), levels=levels)
+    conf = ax.pcolormesh(K[:,:,0], K[:,:,1], SdB, cmap=cm.plasma, vmin=0, vmax=np.max(SdB), shading='giraud')
```

Comment:
what are the advantages of this plot?

Reply:
Fewer interpolation artifacts. Easier to see the sidelobe structure

Reply:
Acceptable change! :+1: